### PR TITLE
prettier english

### PR DESCRIPTION
**Pull request by fobos531**
Here's a version of index.html with improved English. I've included a few comments so feel free to delete those.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -85,21 +85,21 @@ <h1 class="boxy"><a class="anchor" id="about">About me</a></h1>
...
 				techniques such as color coding are. Content structuring, color coding helped content engagement and interaction grow
-				significantly. Im happy that my work reaches over <strong>1 million unique users </strong>every month on Windows community
+				significantly. I'm happy that my work reaches over <strong>1 million unique users </strong>every month on Windows community
 				alone while on /r/Olympics community during 2016 Olympic games my work was seen by <strong>3 million unique users </strong>and
 				viewed <strong>22 million times.</strong> I've received job offers from people that loved my work on <a href="https://windows10.reddit.com"
-				    target="_blank">Windows Community</a> and other communities also thanks to this side project I have had plenty of interaction
-				with Microsoft engineers, prominent developers and lots of other interesting people.
+				    target="_blank">Windows Community</a> and other communities as well. Thanks to this side project, I have had plenty of interaction
+				with Microsoft engineers, prominent developers and lots of other interessting people.
```
Comment (Jasius) [inline index.html]: interesting 🤔

File: index.html [hunk 2]
```diff
@@ -85,21 +85,21 @@ <h1 class="boxy"><a class="anchor" id="about">About me</a></h1>
 						today = new Date,
 						age = today.getFullYear() - year;
 					(today.getMonth() < month || today.getMonth() == month && today.getDate() < day) && age--, document.write(age);
-				</script> years old self-taught Front-end developer. Currently working on part time projects and maintaining Reddit community
-				forum during my free time.
+				</script>-years-old self-taught front-end developer. I am currently working on part-time projects and maintaining Reddit community<!-- When using your age as an adjective, hyphens are used. -->
+				forum in my free time.<!-- "During" is used when describing concrete activities (e.g. football). I feel like "in" is more appropriate here.  -->
```
Comment (Jasius) [inline index.html]: Could you remove those three comments? Thanks for the tips.

**Review by Jasius — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (fobos531) [reply index.html]: done

File: index.html (hunk 1 above)
Comment (fobos531) [reply index.html]: woops :D

**Review by Jasius — APPROVED** [review]
Thanks once again! <3

**Comment by fobos531** [comment]
:+1:

**Pull request merged**